_specs/ecip-1000.md

```diff
@@ -172,6 +172,7 @@ There are three types of ECIP:
   - **Interface** - improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names  and contract ABIs.
   - **ECBP (Ethereum Classic Best Practice)** - application-level standards and conventions, including contract standards such as token standards, name registries, URI schemes, library/package formats, and wallet formats.
 - A **Meta ECIP** describes a **process** surrounding Ethereum Classic or proposes a change to (or an event in) a process. Process ECIPs are like Standard Track ECIPs, but apply to areas other than the Ethereum Classic protocol itself. They may propose an implementation, but not to Ethereum Classic's codebase; they often require community consensus; unlike Informational ECIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process (e.g. this ECIP-1000 process document), and changes to the tools or environment used in Ethereum Classic development. *Any meta-ECIP is also considered a Process ECIP.*
+- A **Fork ECIP** describes a feature set and a block activation number for those features to become active on one or more blockchains.
```

Comment:
This should reference a link to ECIP-1077.

Reply:
Resolved with https://github.com/ethereumclassic/ECIPs/pull/224/commits/b9478e610fdf1ae082ec32c0d9d4408ff5ced481